### DISCUSSION TITLE
Hey can I customize the code block component and modify it a little bit for some files.

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#952] [Q&A] [answered]
This is what the current code block looks like 
![image](https://github.com/user-attachments/assets/5e7347e4-3e05-4818-a326-1b06a8ed0bf9)

what I want is to add some buttons to this codeblock if possible without creating a new component. 

![image](https://github.com/user-attachments/assets/64a46123-d930-4ec6-aa17-38b503963ba0) and I will add some functionality to it. 

PS: Sorry for not contributing or pr, you had already added and modified the documentation component in the navigation bar, don't worry I am looking forward to new design or development ideas.

**Comment by fuma-nama** [comment, marked as answer]
The code block component is what you need (see docs for details)
